zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/zeppelinhub/rest/ZeppelinhubRestApiHandler.java

```diff
@@ -128,17 +128,17 @@ public String asyncGet(String argument) throws IOException {
       response = listener.get(30, TimeUnit.SECONDS);
     } catch (InterruptedException | TimeoutException | ExecutionException e) {
       LOG.error("Cannot perform Get request to ZeppelinHub", e);
-      throw new IOException("Cannot load note from ZeppelinHub", e);
+      throw new IOException("Cannot perform Get request to ZeppelinHub", e);
     }
 
     int code = response.getStatus();
     if (code == 200) {
-      try (InputStream responseContent = listener.getInputStream()) {
```

Comment:
here [try-with-resource statements](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) (and [here](https://www.mkyong.com/java/try-with-resources-example-in-jdk-7/)) is automatically closable, and it's syntactic sugar for what you did, so basically they're equivalent and no need to change